events/importer/enkora.py

```diff
@@ -1784,16 +1809,28 @@ def keyword_ids_into_keywords(kw_ids: set) -> list[Keyword]:
 
         return kws
 
-    def convert_title(self, course: dict) -> tuple[str, str]:
+    @staticmethod
+    def convert_title(course: dict) -> tuple[dict, Optional[dict]]:
         """
         Add information to course title about event hour and weekday
         :param course: dict, Enkora course information as returned from API
-        :return: tuple[str, str], modified course title
+        :return: tuple[dict, Optional[dict]], dictionary of course titles
+        and modified course titles in supported languages
         """
-        course_title = course["reservation_event_group_name"].strip()
+        course_title = {}
+
+        for lang in EnkoraImporter.supported_languages:
+            course_title.update(
+                {lang: course.get(f"reservation_event_group_name_{lang}")}
+            )
```

Comment:
```suggestion
        course_title = {
            lang: course.get(f"reservation_event_group_name_{lang}")
            for lang in EnkoraImporter.supported_languages
        }
```

would be more succinct and Pythonic.

Reply:
done